mlflow/pyfunc/__init__.py

```diff
@@ -1745,6 +1747,47 @@ def _prebuild_env_internal(local_model_path, archive_name, save_path):
         shutil.rmtree(env_root_dir, ignore_errors=True)
 
 
+def _download_prebuilt_env_if_needed(prebuilt_env_path):
+    from databricks.sdk import WorkspaceClient
+
+    from mlflow.utils.file_utils import get_or_create_tmp_dir
+
+    parsed_url = urlparse(prebuilt_env_path)
+    if parsed_url.scheme == "" or parsed_url.scheme == "file":
+        # local path
+        return parsed_url.path
+    if parsed_url.scheme == "dbfs":
+        tmp_dir = MLFLOW_MODEL_ENV_DOWNLOADING_TEMP_DIR.get() or tempfile.mkdtemp(
+            dir=get_or_create_tmp_dir()
+        )
```

Comment:
Why do we need to call `tempfile.mkdtemp`? `get_or_create_tmp_dir` doesn't create a temporary directory?